roles/development/files/bin/make-changelog.py

```diff
@@ -106,44 +153,78 @@ def main(username, project, version):
         relstr = "Merge branch 'release"
         pullstr = "Merge pull request #"
 
-        commits = [(slug, msg) for slug, msg in commits if relstr not in msg]
-        pulls = [(slug, msg) for slug, msg in commits if pullstr in msg]
-        commits = [(slug, msg) for slug, msg in commits if pullstr not in msg]
+        if pagure:
+            pullstr = "Merge #"
+
+        commits = [
+            (slug, comdate, msg)
+            for slug, comdate, msg in commits
+            if relstr not in msg]
+        pulls = [
+            (slug, msg)
+            for slug, comdate, msg in commits
+            if pullstr in msg]
+        commits = [
+            (slug, comdate, msg)
+            for slug, comdate, msg in commits
+            if pullstr not in msg]
 
         print
         print start
         print "-" * len(start)
 
+        if pagure:
+            pulls.extend(get_pull_request_pagure(project, commits[0]))
```

Comment:
I want the date of the first commit after the last tag, so we get only the pull-request merged after that commit